PowerUp/app/src/main/java/powerup/systers/com/GameActivity.java

```diff
@@ -218,4 +222,22 @@ public DatabaseHandler getmDbHandler() {
 	public void setmDbHandler(DatabaseHandler mDbHandler) {
 		this.mDbHandler = mDbHandler;
 	}
+
+    @Override
+    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+
+        if(requestCode == 1){
```

Comment:
Please run the code formatter on Code->Reformat Code. There should be a space before and after brackets in the if